README.md

```diff
@@ -181,30 +181,51 @@ See [Building](https://github.com/rook/rook/wiki/Building) in the wiki for more
 
 ## Design
 
-A rook cluster is made up of one or more nodes each running the Rook daemon `rookd`. Containers and Pods can
-mount block devices and filesystems exposed by the cluster, or can use S3/Swift API for object storage. There is
-also a REST API exposed by `rookd` as well as a command line tool called `rook`.
+Rook is fully integrated with Kubernetes. Once Rook is running in the Kubernetes cluster, Kubernetes applications can
```

Comment:
You might strike the first sentence.  I'm not sure I know what "fully integrated" means.